S/SPRAL/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SPRAL"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ralna/spral.git", "cd2d2e817275f16d586cf72767631b3c7472ce02")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/spral
+if [[ "${target}" == *-freebsd* ]] || [[ "${target}" == *-apple-* ]]; then
+    CC=gcc
+    CXX=g++
+fi
+./autogen.sh
+mkdir build
+cd build
+CFLAGS=-fPIC CPPFLAGS=-fPIC CXXFLAGS=-fPIC FFLAGS=-fPIC FCFLAGS=-fPIC \
+    ../configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
+    --with-blas="-L${libdir} -lopenblas" --with-lapack="-L${libdir} -lopenblas" \
+    --with-metis="-L${libdir} -lmetis" --with-metis-inc-dir="${prefix}/include"
+make
+gfortran -fPIC -shared -Wl,$(flagon --whole-archive) libspral.a -Wl,$(flagon --no-whole-archive) -lgomp -lopenblas -lhwloc -lmetis -lstdc++ -o libspral.${dlext}
+make install
+cp libspral.${dlext} ${libdir}
+exit
```

Comment:
https://github.com/JuliaPackaging/Yggdrasil/blob/8be19820afa24ff817185f3c53c4d20786a8add3/S/SHTOOLS/build_tarballs.jl#L28

```suggestion
gfortran -fPIC -shared -Wl,$(flagon --whole-archive) libspral.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lgomp -lopenblas -lhwloc -lmetis -lstdc++ -o ${libdir}/libspral.${dlext}
make install
```